assignment/core/src/main/java/com/assignment/core/workflow/CSVpageCreationProcess.java

```diff
@@ -0,0 +1,221 @@
+package com.assignment.core.workflow;
```

Comment:
How will this run on the AEM instance? Add workflow model design and runtime model in code.